ComputeLibrary/Source/arm_cl_tables.c

```diff
@@ -39,6 +39,18 @@ const float32_t exp_tab[4*8] =
         0.000195780929062f,0.000195780929062f,0.000195780929062f,0.000195780929062f
 };
 
+const float64_t exp_tab_64[2*8] =
+{
+        1.f,1.f,
+        0.0416598916054f,0.0416598916054f,
```

Comment:
The constant are float (f suffix) for a float64 array.
It would be better to use float64.
I unfortunately don't know which formula was used to generate the values. So at least, 
removing the f should be enough. Ideally, regenerating the array with float64 values may be even better for accuracy.